tests/routes/patch_mediation_test.py

```diff
@@ -38,6 +38,27 @@ def when_mediation_is_edited(self, app):
             assert response.json['thumbUrl'] == mediation.thumbUrl
             assert mediation.isActive == data['isActive']
 
+        @pytest.mark.usefixtures("db_session")
+        def should_match_a_mediation_description(self, app):
+            # given
+            user = create_user()
+            offerer = create_offerer()
+            venue = create_venue(offerer)
+            offer = create_offer_with_event_product(venue)
+            user_offerer = create_user_offerer(user, offerer)
+            mediation = create_mediation(offer)
+            repository.save(mediation)
+            repository.save(user, venue, offerer, user_offerer)
+            auth_request = TestClient(app.test_client()).with_auth(email=user.email)
+            data = {'isActive': False}
+
+            # when
+            response = auth_request.patch('/mediations/%s' % humanize(mediation.id), json=data)
+
+            # then
+            assert response.status_code == 200
+            assert [*response.json] == ['authorId', 'credit', 'dateCreated', 'dateModifiedAtLastProvider', 'fieldsUpdated', 'id', 'idAtProviders', 'isActive', 'lastProviderId', 'offerId', 'thumbCount', 'thumbUrl']
```

Comment:
Si vraiment on veut ce test (cf. mon commentaire au-dessus), je suggère plutôt de comparer les clés comme suit, pour éviter de dépendre de l'ordre des clés d'un dictionnaire :

```suggestion
            assert set(response.json) == {
                'authorId', 'credit', 'dateCreated', 'dateModifiedAtLastProvider', 'fieldsUpdated', 'id',
                'idAtProviders', 'isActive', 'lastProviderId', 'offerId', 'thumbCount', 'thumbUrl'
            }
```

Reply:
En fait, on veux autant tester le contenu que la structure des données.
Ici, l'objectif de ce test `should_match_a_mediation_description` est de décrire et sécuriser ce qu'est une médiation aujourd'hui. "Voilà le contrat d'interface" en quelque sorte.

Reply:
J'ai ajouté un test sur les données au niveau du get, mais si ça te rassure, j'ajoute un test pour les données qu'on pourra eventuellement virer après le refacto